[JCR Importer] generated xml triggers a SAXParseException

The following generated xml triggers an xml Exception when the package is installed in AEM:
```
keywords=\"[Automation,mRNA,Precision & Genomic Medicine,Monoclonal Antibodies]\" topics=\"Automation,mRNA,Precision & Genomic Medicine,Monoclonal Antibodies\"
```

Error in AEM:
```
Caused by: org.xml.sax.SAXParseException: The entity name must immediately follow the '&' in the entity reference.
	at java.xml/com.sun.org.apache.xerces.internal.util.ErrorHandlerWrapper.createSAXParseException(ErrorHandlerWrapper.java:204)
```

 '& ' should be escaped: replacing '& ' with '&amp; ' fixes the error.

Using following page metadata config:
```
  {
    "id": "page-metadata",
    "fields": [
      {
        "component": "container",
        "label": "Fieldset",
        "fields": [
          {
            "component": "text",
            "valueType": "string",
            "name": "jcr:title",
            "label": "Title"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "jcr:description",
            "label": "Description"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "cq:canonicalUrl",
            "label": "Canonical"
          },
          {
            "component": "aem-tag",
            "valueType": "string",
            "name": "cq:tags",
            "multi": true,
            "label": "AEM Tag Picker"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "keywords",
            "multi": true,
            "label": "Keywords"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "topics",
            "multi": true,
            "label": "Topics"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "authorName",
            "label": "Author Name"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "authorTitle",
            "label": "Author Title"
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "brand",
            "label": "Brand"
          },
          {
            "component": "text",
            "valueType": "number",
            "name": "readingTime",
            "label": "Reading Time"
          },
          {
            "component": "date-time",
            "valueType": "date",
            "name": "publishDate",
            "label": "Publish Date",
            "placeholder": "YYYY-MM-DD",
            "displayFormat": null,
            "valueFormat": null
          },
          {
            "component": "text",
            "valueType": "string",
            "name": "navTitle",
            "label": "Navigation Title"
          }
        ]
      }
    ]
  },
```

--- a/.gitignore
+++ b/.gitignore
@@ -8,5 +8,5 @@ logs
 .DS_Store
 test-results.xml
 .env
-.node-version
 lib
+.node-version

--- a/src/importer/html2jcr/hast2jcr/handlers/metadata.js
+++ b/src/importer/html2jcr/hast2jcr/handlers/metadata.js
@@ -12,7 +12,7 @@
 
 import { select } from 'hast-util-select';
 import { toString } from 'hast-util-to-string';
-import { toMetaName } from '../utils.js';
+import { encodeHTMLEntities, toMetaName } from '../utils.js';
 
 const metadataFields = ['description'];
 const aemMapping = {
@@ -52,9 +52,9 @@ const metadata = {
       if (metaNameFields.indexOf(name) === -1 && metaNameFields.indexOf(property) === -1) {
         return acc;
       }
-      let finalContent = content;
+      let finalContent = encodeHTMLEntities(content);
       if (name === 'robots' || name === 'keywords') {
-        finalContent = content.split(',').map((value) => value.trim());
+        finalContent = content.split(',').map((value) => encodeHTMLEntities(value.trim()));
         finalContent = finalContent.length > 0 ? `[${finalContent.join(',')}]` : '';
       }
       for (const [key, value] of Object.entries(aemMapping)) {

--- a/test/importers/html2jcr/fixtures/metadata.expected.xml
+++ b/test/importers/html2jcr/fixtures/metadata.expected.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
-    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Sites Franklin Example" template="article-template" jcr:description="Test Description" abstract="Lorem Ipsum">
+    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Sites Franklin Example" template="article-template" jcr:description="Test Description" abstract="Lorem &amp; Ipsum">
         <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
             <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                 <title sling:resourceType="core/franklin/components/title/v1/title" jcr:primaryType="nt:unstructured" jcr:title="Lorem ipsum" type="h2"/>

--- a/test/importers/html2jcr/fixtures/metadata.md
+++ b/test/importers/html2jcr/fixtures/metadata.md
@@ -11,5 +11,5 @@ Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor i
 +---------------+------------------------------------+
 | Description   | Test Description                   |
 +---------------+------------------------------------+
-| Abstract      | Lorem Ipsum                        |
+| Abstract      | Lorem & Ipsum                      |
 +---------------+------------------------------------+

--- a/test/importers/html2jcr/fixtures/page-metadata.expected.xml
+++ b/test/importers/html2jcr/fixtures/page-metadata.expected.xml
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
-    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Test Page" jcr:description="Test description" datetime="2024-04-23T15:53:00.000+02:00" cq:robotsTags="[index,follow]" keywords="[foo,bar]" authorTitle="John Doe" abstract="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eu purus id tellus consequat pharetra. Fusce blandit felis ut dui congue, cursus suscipit est hendrerit." template="salescontent">
+    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page" jcr:title="Test Page" jcr:description="Test description" datetime="2024-04-23T15:53:00.000+02:00" cq:robotsTags="[index,follow]" keywords="[foo,bar,lorem &amp; ipsum]" authorTitle="John Doe" abstract="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eu purus id tellus consequat pharetra. Fusce blandit felis ut dui congue, cursus suscipit est hendrerit." template="salescontent">
         <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root"/>
     </jcr:content>
 </jcr:root>

--- a/test/importers/html2jcr/fixtures/page-metadata.html
+++ b/test/importers/html2jcr/fixtures/page-metadata.html
@@ -7,7 +7,7 @@
     <meta name="description" content="Test description">
     <meta name="datetime" content="2024-04-23T15:53:00.000+02:00"/>
     <meta name="robots" content="index, follow"/>
-    <meta name="keywords" content="foo, bar"/>
+    <meta name="keywords" content="foo, bar, lorem & ipsum"/>
     <meta name="authortitle" content="John Doe"/>
     <meta name="abstract" content="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla eu purus id tellus consequat pharetra. Fusce blandit felis ut dui congue, cursus suscipit est hendrerit."/>
     <meta property="og:title" content="Test Page">

